src/renderer/components/table/table-cell.tsx

```diff
@@ -52,20 +53,20 @@ export class TableCell extends React.Component<TableCellProps> {
     const { checkbox, isChecked } = this.props;
     const showCheckbox = isChecked !== undefined;
     if (checkbox && showCheckbox) {
-      return <Checkbox value={isChecked}/>
+      return <Checkbox value={isChecked} />
     }
   }
 
   render() {
-    const { className, checkbox, isChecked, sortBy, _sort, _sorting, _nowrap, children, title, ...cellProps } = this.props;
+    const { className, checkbox, isChecked, sortBy, _sort, _sorting, _nowrap, children, title, renderBoolean: displayBoolean, ...cellProps } = this.props;
     const classNames = cssNames("TableCell", className, {
       checkbox: checkbox,
       nowrap: _nowrap,
       sorting: this.isSortable,
     });
-    const content = title || children;
+    const content = displayBooleans(displayBoolean, title || children)
```

Comment:
I would suggest to explicitly check for a boolean here without using an external wrapper and utility